aws_controller.py

```diff
@@ -38,3 +38,31 @@ def get_trails_by_length(data):
     items = response.get('Items', [])
     items=sort_items(items,'length',True)
     return items[:15]
+
+def get_trails_by_rating(data):
+    state = data.get('state', None)
+    rating = data.get('rating', None)
```

Comment:
provide default value to rating when rating is not passed. Default it to 5, hence eliminating rating is none condition.

Reply:
done